keps/sig-network/20200103-mixed-protocol-lb/README.md

```diff
@@ -1,28 +1,4 @@
----
-title: Different protocols in the same Service definition with type=LoadBalancer
-authors:
-  - "@janosi"
-owning-sig: sig-network
-participating-sigs:
-  - sig-cloud-provider
-reviewers:
-  - "@thockin"
-  - "@dcbw"
-  - "@andrewsykim"
-approvers:
-  - "@thockin"
-editor: TBD
-creation-date: 2020-01-03
-last-updated: 2020-07-09
-status: provisional
-see-also:
-replaces:
-superseded-by:
----
-
-# different protocols in the same service definition with type=loadbalancer
-
-## Table of Contents
+# KEP-NNNN: different protocols in the same service definition with type=loadbalancer
```

Comment:
Those `N's` should be replaced by the actual issue number this KEP is addressing. i.e: `KEP-1435` which corresponds to your issue: https://github.com/kubernetes/enhancements/issues/1435.

Reply:
Fixed.